File: ITSC 3155/itsc-3155/m1/syntax/loops/whiles.java
// basic while loops in Java
package m1.syntax.loops;

public class whiles {
    public static void main(String[] args) {
        int x = 0;
        while (x < 10) {
            System.out.println("Looping: x is " + x);
            x *= 2;
        }
        System.out.println("x is now " + x);
    }

}

File: ITSC 3155/itsc-3155/m1/syntax/loops/whiles.py
# basic while loops in Python

def main():
    x = 0

    while x < 10:
        # f-strings allow for easy string formatting
        print(f"Looping: x is {x}")
        x *= 2  # this is the same as x = x * 2

    print(f"x is now {x}")


if __name__ == "__main__":
    main()
